plugins/modules/sensu_go_asset.py

```diff
@@ -26,7 +30,7 @@
     description:
       - Target state of the Sensu object.
     type: str
-    choices: [ 'present' ]
+    choices: [ "present", "absent" ]
```

Comment:
Good find, feature to delete asset was only merged 24 days ago and in https://docs.sensu.io/sensu-go/5.12/release-notes/#5-12-0-release-notes I can see it's released already 👍

Reply:
It was not me who fount this; it was a sanity check that reported a mismatch between argument specification and documentation. It was pure luck that I "fixed" this in the right direction ;)

Reply:
Haha, I like your lucky hand